doc/uftrace-record.md

```diff
@@ -875,6 +879,121 @@ Each field in 'script_context' can be read inside the script.
 Please see `uftrace-script`(1) for details about scripting.
 
 
+AGENT
+=====
```

Comment:
I think this section belongs to the `uftrace live`.  Maybe we can just refer to it instead of having a copy here.

"Please see `uftrace-live`(1) for details about agent operations."